FreydCategoriesForCAP/gap/FreydCategory.gi

```diff
@@ -1537,22 +1537,27 @@ InstallMethod( Display,
                [ IsFreydCategoryMorphism ],
                
   function( freyd_category_morphism )
-
-    Print( Concatenation( StringMutable( freyd_category_morphism ), "\n\n" ) );
-    Print( "--------------------------------" );
-
-    Print( Concatenation( "\n\n", "Source:\n" ) );
+    
+    Print( Concatenation( "\n", "--------------------------------\n" ) );
+    Print( "Source:\n" );
+    Print( "--------------------------------\n\n" );
     Display( RelationMorphism( Source( freyd_category_morphism ) ) );
-    Print( "--------------------------------" );
+    Print( Concatenation( "\n\n", "--------------------------------", "\n" ) );
     
-    Print( Concatenation( "\n\n", "Morphism datum:\n" ) );
+    Print( "Morphism datum:\n" );
+    Print( "--------------------------------\n\n" );
     Display( MorphismDatum( freyd_category_morphism ) );
-    Print( "--------------------------------" );
+    Print( Concatenation( "\n\n", "--------------------------------", "\n" ) );
     
-    Print( Concatenation( "\n\n", "Range:\n" ) );
+    Print( "Range:\n" );
+    Print( "--------------------------------\n\n" );
     Display( RelationMorphism( Range( freyd_category_morphism ) ) );
-    Print( "--------------------------------" );
-
+    Print( Concatenation( "\n\n", "--------------------------------", "\n" ) );
+    
+    Print( "General data:\n" );
```

Comment:
```suggestion
    Print( "General description:\n" );
```
I would rather like to have a clean distinction between this and the term Morphism *datum*.